tmpfiles.sh

```diff
@@ -253,7 +401,12 @@ _D() {
 	local path=$1 mode=$2 uid=$3 gid=$4
 
 	if [ -d "${path}" ] && [ "${REMOVE}" -gt 0 ]; then
-		dryrun_or_real find "${path}" -mindepth 1 -maxdepth 1 -xdev -exec rm -rf {} +
+		if owned_by_root "${path}" ; then
```

Comment:
can't tmpfiles.d rules create dirs not owned by root by design ?  so this basically wedges the state after a single run ?

Reply:
Directory creation is refused only when a directory with the same name, owned by root already exists.
Removing directories owned by root creates potential a security risk.
Changing ownership of directories owned by root creates potential a security risk.
Only the system administrator should be able to knowingly remove directories owned by root or change ownership.